src/cli/test/ionic-config.spec.ts

```diff
@@ -0,0 +1,58 @@
+import fs from 'fs-extra';
+import { readConfig, writeConfig, updateConfig, DEFAULT_CONFIG_DIRECTORY } from '../ionic-config';
+
+describe('readConfig', () => {
+
+	it('should create a file if it does not exist', async () => {
+		if (await fs.pathExists(DEFAULT_CONFIG_DIRECTORY)) {
+			fs.rmSync(DEFAULT_CONFIG_DIRECTORY);
+		}
+
+		expect(await fs.pathExists(DEFAULT_CONFIG_DIRECTORY)).toBe(false)
+
+		const config = await readConfig();
+
+		expect(typeof config).toBe("object");
```

Comment:
Same here, I don't think we strictly need these assertions on the type of an object